google-cloud-asset/README.md

```diff
@@ -28,7 +28,7 @@ $ gem install google-cloud-asset
   to see the full list of Cloud APIs that we cover.
 
 [Client Library Documentation]: https://googleapis.dev/ruby/google-cloud-asset/latest
-[Product Documentation]: https://cloud.google.com/asset
+[Product Documentation]: https://cloud.google.com/resource-manager
```

Comment:
Is this change correct? This URL looks to cover the API that is used by google-cloud-resource_manager, which is REST only. The google-cloud-asset is a different API and uses a gRPC endpoint.

Reply:
That is really weird. Might be a config error.

Reply:
Okay, after investigating this, cloud asset is part of the resource manager, and the new link is actually correct. (In fact, the old link doesn't exist and leads to a 404.) So this change is correct.